### PR TITLE
Fix formatting issues in torch.tensor_split documentation

### PR DESCRIPTION
**Pull request by kurtamohler**

### CODE REVIEW

File: torch/_torch_docs.py [hunk 1]
```diff
@@ -1239,11 +1239,8 @@ def merge_dicts(*dicts):
 
         If :attr:`indices_or_sections` is a list of ints, :attr:`input` is split along
         dimension :attr:`dim` at each of the indices in the list. For instance,
-        :code:`[2, 3]` and :code:`dim=0` would result in the following tensors:
-
-            - :code:`input[:2]`
-            - :code:`input[2:3]`
-            - :code:`input[3:]`
+        :code:`[2, 3]` and :code:`dim=0` would result in the tensors :code:`input[:2]`,
```
Comment (mruberry) [inline torch/_torch_docs.py:1242]: Maybe `indices_or_sections=[2, 3]` in the first code block? Your call. Just pint me with your thinking.

**Review by mruberry — APPROVED** [review]
Thanks @kurtamohler! Just have one question for you. Ping me when you'd like this merged.

File: torch/_torch_docs.py (hunk 1 above)
Comment (kurtamohler) [reply torch/_torch_docs.py]: Yeah that makes sense. Updated.

**Comment by kurtamohler** [comment]
@mruberry, I think this is good now.

**Pull request closed without merging**